composer.lock

```diff
@@ -4,26 +4,26 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "db9c65c6be1db5f7ffb939471c12fade",
+    "content-hash": "3cd2d8a2bbe505621c24a218d78894d7",
     "packages": [
         {
             "name": "laminas/laminas-cache",
-            "version": "3.8.0",
+            "version": "3.10.1",
```

Comment:
Let's upgrade just the minimum necessary: the fact that you changed the whole `composer.lock` led to the amount of changes you had to adjust afterwards, while #52 manages that separately

Reply:
Resolved